lib/sm/client.rb

```diff
@@ -163,7 +163,9 @@ def post_create_message_draft(args = {})
       validate_draft(args)
 
       json = perform(:post, 'message/draft', args, token_headers).body
-      MessageDraft.new(json)
+      draft = MessageDraft.new(json)
+      draft.body = draft.instance_variable_get(:@original_attributes)[:body]
```

Comment:
I think we could expose the body through a method or an attr_reader. That would avoid using `instance_variable_get` which is usually reserved for working around access to attributes in code you can't modify (e.g. inside a gem).

Reply:
gotcha, will do